### ISSUE TITLE
Spell check in \newcommand doesn't work

### ISSUE THREAD

**Issue by andhla** [texstudio-org/texstudio#447]
### Environment
- TeXstudio: 2.12.14
- Qt: 5.8.0
- OS: Windows 7
- TeX distribution: miktex

### Actual behavior
Spell check in \newcommand doesn't work.

![img](https://user-images.githubusercontent.com/46059350/50367662-15bb6200-0582-11e9-96c7-df05281b80a0.jpg)

**Comment by timhoffm** [comment]
@sunderme I think this is not what the issue is about.

The contents of
~~~
\newcommand{\test}{
stuff-here
}
~~~
is essentially the same as writing `stuff-here` directly in the .tex file. So spellchecking should be active for `stuff-here`. Not sure if that's possible.

**Comment by sunderme** [comment]
no, it should not and it is not. Usually a lot of strange things happen in command definitions, so syntax and spellchecking is disabled there.

**Comment by andhla** [comment]
I solve my issue.
I define: \newcommand{\mynewcommand}[2]{\newcommand{#1}{#2}} (+ cwl file )
and use \mynewcommand spellchecking work.

I use it like this:
\mynewcommand{\groupA}{exercises}
\mynewcommand{\groupB}{exercises}

\begin{document}
\groupA \groupA \groupB \groupB
\end{document}

Thanks
